### PR TITLE
adds itemNew code to send item objects to app and to show a list of t…

### PR DESCRIPTION
**Pull request by elysemontano**
adds itemNew code to send item objects to app and to show a list of the items added to the item new page

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -22,12 +22,19 @@ class App extends React.Component {
     this.state = {
       parties: MockParties,
       items: MockItems,
+      //mock party id
+      party_id: 0,
```
Comment (minachuong) [inline app/javascript/components/App.js]: Nitpick: general practice is to store values initializing values as undefined

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -22,12 +22,19 @@ class App extends React.Component {
...
     };
   }
 
   createNewEvent = (newEvent) => {
     console.log(newEvent);
-  }
+    this.setState({ party_id: (this.state.party_id += 1) });
```
Comment (minachuong) [inline app/javascript/components/App.js]: this logic seems to be dependent on mock data and not data that will be actually represented in the database

**Review by minachuong — COMMENTED** [review]
generally mockdata should represent real data, so there should be no logic added that only supports mockdata. Please see me if this is unclear feedback - Mina

**Comment by elysemontano** [comment]
We started integrating frontend to backend to resolve the logic in mock data.  Code is not 100%, but we need to update main for both this branch and rspec branch to move forward.

**Pull request merged**